content/sensu-core/2.0/guides/clustering.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Clustering"
+linkTitle: "Clustering"
+weight: 120
+version: "2.0"
+product: "Sensu Core"
+platformContent: false
+menu: 
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What is a Sensu Cluster?
+
+A Sensu Cluster is a group of [at least 3][1] sensu-backend nodes, each supporting an embedded or external etcd instance. Creating a Sensu Cluster ultimately configures an [etcd cluster][2].
```

Comment:
The word "supporting" throws me off here. I'm not an expert on etcd by any means, but could/should this read "each connected to a shared etcd cluster, using Sensu's embedded etcd or an external etcd cluster" instead of "each supporting an embedded or external etcd instance"?

Reply:
Love that improvement!